docs/docs/rest-api/public/api/v2/groups.raml

```diff
@@ -0,0 +1,376 @@
+###
+# Get the root group
+#
+get:
+  description:
+    Get the root group with all applications and all transitive child groups.
+  is: [ secured ]
+  responses:
+    200:
+      description: The root group with all transitive dependencies.
+      body:
+        application/json:
+          example: !include examples/group.json
+          schema: Group
+
+###
+# Update the root group
+#
+put:
+  description:
+    Change parameters of a deployed application group.
+    The new group parameters get applied.
+
+
+    * Changes to application parameters will result in a restart of this application.
+
+    * A new application added to the group is started.
+
+    * An existing application removed from the group gets stopped.
+
+
+    If there are no changes to the application definition, no restart is triggered.
+    During restart marathon keeps track, that the configured amount of minimal running instances are _always_ available.
+
+    This method alows 2 special modes for the update operation>
+
+    * Provide only the `version` field in the group definition. This will rollback the group to that given version
+
+    * Provide only the `scaleBy` field will scale all transitive applications by the given factor.
```

Comment:
s/applications/applications' instance counts

Also: an explanation how rounding will be applied might be helpful. Imagine I have a group with 3 apps:
- app-1, instance count: 100
- app-2, instance count: 100
- app-3, instance count: 1

people will ask (themselves or us): what happens to app-3 when I scaleBy 0.1?